packages/core/helper-plugin/package.json

```diff
@@ -64,16 +71,33 @@
     "@storybook/builder-webpack5": "6.5.15",
     "@storybook/manager-webpack5": "6.5.15",
     "@storybook/react": "^6.5.10",
-    "@strapi/design-system": "1.4.1",
-    "@strapi/icons": "1.4.1",
     "@testing-library/react": "12.1.4",
     "@testing-library/react-hooks": "8.0.1",
     "cross-env": "^7.0.3",
     "esbuild-loader": "^2.20.0",
-    "react-test-renderer": "^17.0.2",
-    "require-from-string": "2.0.2",
+    "match-sorter": "^4.0.2",
     "rimraf": "3.0.2",
-    "typescript": "4.6.2"
+    "typescript": "4.6.2",
+    "webpack": "^5.74.0"
+  },
+  "peerDependencies": {
+    "@strapi/design-system": "1.4.1",
+    "@strapi/icons": "1.4.1",
```

Comment:
@joshuaellis I guess these dependencies shouldn't show up in both dependencies and peerDependencies?

If there is a chance to isolate the the actual changes from the deps changes I'd prefer that to limit the risk of this PR a bit.

Reply:
🪄